pkg/conf/flags.go

```diff
@@ -45,3 +41,21 @@ var CassandraSslCertPath = NewStringFlag("cassandra_ssl_cert_path", "Sets the cl
 
 // CassandraSslKeyPath sets the client private key, in case the cluster requires client verification.
 var CassandraSslKeyPath = NewStringFlag("cassandra_ssl_key_path", "Sets the client private key, in case the cluster requires client verification.", "")
+
+// CassandraPort sets port of Cassandra DB endpoint.
+var CassandraPort = NewIntFlag("cassandra_port", "Port of Cassandra DB endpoint.", 9042)
+
+// CassandraTimeout sets query timeout for the publisher in seconds.
+var CassandraTimeout = NewIntFlag("cassandra_timeout", "Query timeout for the publisher in seconds.", 0)
+
+// CassandraInitialHostLookup sets that driver will attempt to get host info from Cassandra.
+var CassandraInitialHostLookup = NewBoolFlag("cassandra_initial_host_loopup", "If false, driver won't attempt to get host info from Cassandra.", true)
```

Comment:
Could you elaborate, in what situation, getting host info from Cassandra is necessary?